services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -135,9 +135,6 @@ export class NOWProgressActions extends Component {
       .updateNoticeOfWorkApplicationProgress(
         this.props.noticeOfWork.now_application_guid,
         tab,
-        {
-          end_date: new Date(),
-        },
         message
```

Comment:
does `updateNoticeOfWorkApplicationProgress` need to be updated to reference params differently? 
`message` is now in the `payload` position

Reply:
re added the payload.